index.html

```diff
@@ -4,306 +4,353 @@
-            <li class="language3">Ruby</li>
-          </ul>
-          <button>See Project</button>
-        </div>
-
-        <div class="work1" id="w4">
-          <h1>Profesional Art Printing Data</h1>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard
-          </p>
-          <ul class="category">
-            <li class="language1">html</li>
-            <li class="language2">bootstrap</li>
-            <li class="language3">Ruby</li>
-          </ul>
-          <button>See Project</button>
-        </div>
-
-        <div class="work1" id="w5">
-          <h1>Profesional Art Printing Data</h1>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard
-          </p>
-          <ul class="category">
-            <li class="language1">html</li>
-            <li class="language2">bootstrap</li>
-            <li class="language3">Ruby</li>
-          </ul>
-          <button>See Project</button>
+        <div class="MultipostStories">
+          <img src="./img/FeatureImage.png" />
+          <div class="info">
+            <p class="MultipostStories-heading">Multi-Post Stories</p>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul>
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+          </div>
         </div>
-
-        <div class="work1" id="w6">
-          <h1>Profesional Art Printing Data</h1>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard
-          </p>
-          <ul class="category">
-            <li class="language1">html</li>
-            <li class="language2">bootstrap</li>
-            <li class="language3">Ruby</li>
-          </ul>
-          <button>See Project</button>
+        <div class="Works">
+          <div id="w1" class="work1">
+            <h2 class="project">Profesional Art Printing Data</h2>
+            <p class="project">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="project">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="show-button"></div>
+            <button class="btn-porject">See Project</button>
+          </div>
```

Comment:
As requested by the previous reviewer, please add the correct hover effect to your project cards:
Your design:
![image](https://user-images.githubusercontent.com/42799579/193397930-f25361f2-4863-467e-9160-2da4b6718462.png)

Figma design:
![image](https://user-images.githubusercontent.com/42799579/193004940-9a5ae8e6-3116-429e-843c-b470fb4c26d3.png)
If you pay closer attention, you will see that in the design there is a cursor hovering over this first card.
This means that this card isn't different from the other ones, the design is showing to you the _hover_ effect of the cards. 😉
In short, all your cards should have the title/description but, when hovered, the project picture appears clearly and the _See Project_ button is displayed.
Keep up the good work!

Reply:
hi @Gopxfs 
The hover for the first card is working fine on my system.
![image](https://user-images.githubusercontent.com/107232162/193402451-f02a5ae9-9bf5-4f0a-bd72-228ee5ea8c0d.png)
the above image is of the card when hovered.
![image](https://user-images.githubusercontent.com/107232162/193402501-15cb4704-6909-4023-ad7c-6b74a0d0a275.png)
the above image is of the card when not hovered


![comment](https://user-images.githubusercontent.com/107232162/193402648-20859ea2-17eb-46cb-9024-96fbaf6604b5.png)
can you please contact me on slack or mail me at karanjain2212@gmail.com